bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -70,6 +74,9 @@ <h2>{{ _('Fast for good.') }}</h2>
   <ul id="other-platforms-languages-wrapper" class="small-links desktop hidden">
     <li><button id="other-platforms-modal-link" type="button">{{_('Advanced install options & other platforms') }}</button></li>
     <li><a href="{{ url('firefox.all') }}">{{_('Download in another language') }}</a></li>
+  {% if l10n_has_tag('fxnew_account_032019') %}
+    <li><a href="https://support.mozilla.org/kb/refresh-firefox-reset-add-ons-and-settings?utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=fix-problem-link">{{ _('Fix a problem') }}</a></li>
```

Comment:
I notice this new link is shown even to non-Firefox users, is that intended? 

(I'm easy either way tbh as showing it everywhere is simpler, and should cause no harm?)

Reply:
Yeah, since it's just a link to SUMO now I think it should be OK to show to all browsers... but it's also easy to hide if we think that's worth doing?

Reply:
I'm all for keeping things simple, so let's leave it as is for now and we'll see if any issues arise?

Reply:
👍